charts/kubernetes-graphql-gateway/values.yaml

```diff
@@ -0,0 +1,77 @@
+cors:
+  enabled: false
+  allowedHeaders: "*"
+  allowedOrigins: "*"
+
+image:
+  name: ghcr.io/openmfp/kubernetes-graphql-gateway
+  pullPolicy: IfNotPresent
+
+kcp:
+  enabled: true
+
+gateway:
+  port: 8080
+  healthCheck:
+    enabled: true
+    port: 3389
+  metricsPort: 8081
+  graphiql: true
+  logLevel: trace
+  shouldImpersonate: true
+  usernameClaim: email
+
+listener:
+  enabled: true
+  port: 8090
+  healthCheck:
+    enabled: true
+    port: 3390
+  metricsPort: 8091
+
+imagePullSecret: ghcr-credentials
```

Comment:
lets not specify this here, there should be a default comming from common

Reply:
Done!